Source/JavaScriptCore/API/JSRemoteInspector.h

```diff
@@ -47,8 +47,8 @@ JS_EXPORT void JSRemoteInspectorSetParentProcessInformation(JSProcessID, const u
 
 JS_EXPORT void JSRemoteInspectorSetLogToSystemConsole(bool) JSC_API_AVAILABLE(macos(10.11), ios(9.0));
 
-JS_EXPORT bool JSRemoteInspectorGetInspectionEnabledByDefault(void) JSC_API_AVAILABLE(macos(10.11), ios(9.0));
-JS_EXPORT void JSRemoteInspectorSetInspectionEnabledByDefault(bool) JSC_API_AVAILABLE(macos(10.11), ios(9.0));
+JS_EXPORT bool JSRemoteInspectorGetInspectionEnabledByDefault(void) JSC_API_AVAILABLE(macos(10.11), ios(9.0)) JSC_API_DEPRECATED("Use JSGlobalContextIsInspectable on a single JSGlobalContextRef.", macos(10.11, JSC_MAC_TBA), ios(9.0, JSC_IOS_TBA));
```

Comment:
I don't think we keep the `API_AVAILABLE` around when we add a `API_DEPRECATED_WITH_REPLACEMENT`